pages/common/column.md

```diff
@@ -0,0 +1,19 @@
+# column
+
+> Format standard or file input into multiple columns, rows are filled before columns.
```

Comment:
Also, use a `;` instead of `,` to avoid a [run-on sentence](https://en.wikipedia.org/wiki/Run-on_sentence).